mobile/src/splash-screen/splash-screen.tsx

```diff
@@ -91,6 +91,17 @@ class SplashScreenImpl extends React.Component<SplashScreenStateProps & SplashSc
 
     //----------------------------------------------------------------------------
     public render() {
+        const splashContent = this.renderContent();
```

Comment:
Why do you need `splashContent` variable here? Isn't it possible to call `this.renderContent()` inline?

Reply:
Not sure how to do it correctly. Such code
```
<SafeAreaView style={splashScreenStyles.container}>
                {this.renderContent()}
            </SafeAreaView>
```
produce Top-level 'this' expression warning